app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_list = command.split()
+    if len(command_list) == 3:
+        cp = command_list[0]
+        src_file = command_list[1]
+        dst_file = command_list[2]
```

Comment:
```suggestion
        cp, src_file, dst_file = command_list
```
You can unzip it

Reply:
Also rename, because for naw it i not good names:
cp -> cmd
src_file -> src_path
dst_file -> dst_path